pkg/plugins/policies/meshtcproute/plugin/v1alpha1/listeners_clusters.go

```diff
@@ -0,0 +1,67 @@
+package v1alpha1
+
+import (
+	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	api "github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute/api/v1alpha1"
+	plugins_xds "github.com/kumahq/kuma/pkg/plugins/policies/xds"
+	meshroute_xds "github.com/kumahq/kuma/pkg/plugins/policies/xds/meshroute"
+	"github.com/kumahq/kuma/pkg/util/pointer"
+	envoy_common "github.com/kumahq/kuma/pkg/xds/envoy"
+	envoy_tags "github.com/kumahq/kuma/pkg/xds/envoy/tags"
+)
+
+func getClusters(
```

Comment:
Can we reuse `makeHTTPSplit` function? Even though it's has "HTTP" in its name it doesn't look like it actually depends on protocol (so we can rename it to `makeSplit`)

Reply:
The answer is - we could, but it would involve refactoring in the function as splits doesn't contain enough information to build clusters from them (which we need for `tcpProxy` configuration). I decided to go with separate functions and followup with refactoring if necessary in the future.

Reply:
> splits doesn't contain enough information to build clusters from them

What information is missing?

I think `makeHTTPSplit` does exactly the same and adds clusters to `serviceAcc`. So later you can use `[]*plugins_xds.Split` to build TCPProxy and `serviceAcc.Services()` to build clusters.

Reply:
Will work on this in #6911